lib/util.js

```diff
@@ -1,73 +1,23 @@
+const { JSDOM } = require('jsdom');
 /**
  * injectPartial
  * @description takes an html string and injects new html string based on the
  *     set priority and tag location
  */
 
-module.exports.injectPartial = function(base_html, { options, html, priority, location }) {
-
-  let tag;
-  let regex;
-
-  if ( location === 'head' && priority === 'high' ) {
```

Comment:
i think we'll need to re-establish some of the logic here for dynamically injecting the partials after the meta tag